Gemfile.lock

```diff
@@ -401,7 +399,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
I'm not sure why dependabot is doing this 🤔.

I think it's because `nokogiri` still references a specific platform.

If you take a look at https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/4668 there wasn't this issue, and that seems to be the only difference.

Reply:
Looks like it figured itself out 🤷